statistics-reports.md

```diff
@@ -0,0 +1,202 @@
+# Displaying usage statistics
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Statistics for a DSpaceObject
+**GET /api/statistics/usage-reports**
```

Comment:
please don't use the - in the endpoint see https://github.com/DSpace/Rest7Contract/blob/master/README.md#on-the-naming-of-endpoints

Reply:
I've adjusted the syntax. This is a strange design choice since it reduces readability, but this is not a good location to change these names